src/Components/Projects/Projects.tsx

```diff
@@ -50,15 +55,23 @@ const Projects: React.FC = () => {
   };
   // ********to show delete modal*******************
 
-  const showDeleteModal = (itemId: any) => {
+  const showDeleteModal = (itemId: number) => {
     setItemId(itemId);
     setModalState("delete-modal");
   };
   // **********get all projects*****************
-  const getAllProjectsList = () => {
+  const getAllProjectsList = (pageNo: number, title: string) => {
     axios
-      .get(`${baseUrl}/Project/manager`, { headers: requestHeaders })
+      .get(`${baseUrl}/Project/manager`, {
+        headers: requestHeaders,
+        params: {
+          pageSize: 5,
+          pageNumber: pageNo,
+          title: title,
```

Comment:
The backend is not expecting a title query, so you'll always get all the projects.

![image](https://github.com/Rawan-Mohamed/project-management-system-1/assets/64327685/de87a378-e134-4b74-8bc6-39d88f218c2b)